core/src/scss/utilities/functions/z-index/_z-index.scss

```diff
@@ -0,0 +1,18 @@
+@charset "UTF-8";
+
+// z($level)
+//
+// A function to help us get the z-index from the level from the map
+//
+// $level - the name of the z-index level
+//
+// Style guide: Functions.z-index.level
+//
+
+
+@function z($level) {
```

Comment:
Seems straightforward enough. Do you plan on adding a highest or lowest function where they return +1 above the biggest number in $z-levels or -1 of the smallest?

Reply:
Yes, but after looking at examples of that, I didn't see one that actually seemed to work.